azure-iot-device/azure/iot/device/common/mqtt_transport.py

```diff
@@ -115,10 +119,22 @@ def _create_mqtt_client(self):
         """
         logger.info("creating mqtt client")
 
-        # Instantiate client
-        mqtt_client = mqtt.Client(
-            client_id=self._client_id, clean_session=False, protocol=mqtt.MQTTv311
-        )
+        # Instaniate the client
+        if self.websockets:
+            # MQTT Over Websockets
+            mqtt_client = mqtt.Client(
+                client_id=self._client_id,
+                clean_session=False,
+                protocol=mqtt.MQTTv311,
+                transport="websockets",
+            )
+            mqtt_client.ws_set_options(path="/$iothub/websocket")
```

Comment:
>$iothub [](start = 46, length = 7)

is this value for all iothub and dps ?

Reply:
good question. I'm not sure on the DPS path, but it's likely different. Still, this PR will be scoped to just Device Client.